cmd/cluster/aws/create.go

```diff
@@ -0,0 +1,178 @@
+package aws
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/signal"
+	"syscall"
+
+	apifixtures "github.com/openshift/hypershift/api/fixtures"
+	hyperv1 "github.com/openshift/hypershift/api/v1alpha1"
+	"github.com/openshift/hypershift/cmd/cluster/core"
+	awsinfra "github.com/openshift/hypershift/cmd/infra/aws"
+	"github.com/openshift/hypershift/cmd/util"
+	"github.com/spf13/cobra"
+	utilrand "k8s.io/apimachinery/pkg/util/rand"
+)
+
+func NewCreateCommand(opts *core.CreateOptions) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "aws",
+		Short:        "Creates basic functional HostedCluster resources on AWS",
+		SilenceUsage: true,
+	}
+
+	opts.AWSPlatform = core.AWSPlatformOptions{
+		AWSCredentialsFile: "",
+		InfrastructureJSON: "",
+		Region:             "us-east-1",
+		InfraID:            "",
+		InstanceType:       "m4.large",
+		RootVolumeType:     "gp2",
+		RootVolumeSize:     16,
+		RootVolumeIOPS:     0,
+	}
+
+	cmd.Flags().StringVar(&opts.AWSPlatform.AWSCredentialsFile, "aws-creds", opts.AWSPlatform.AWSCredentialsFile, "Path to an AWS credentials file (required)")
+	cmd.Flags().StringVar(&opts.AWSPlatform.InfrastructureJSON, "infra-json", opts.AWSPlatform.InfrastructureJSON, "Path to file containing infrastructure information for the cluster. If not specified, infrastructure will be created")
```

Comment:
infra-json should probably be core which then unmarshals to platform specific infra struct

Reply:
Done.